components/views/login/Login.module.scss

```diff
@@ -0,0 +1,84 @@
+.content {
+    display: grid;
+    place-items: center;
+    height: calc(100vh - var(--nav-height) - var(--footer-height));
+    .formContainer {
+        height: fit-content;
+        width: 20rem;
+        background: rgba(3, 3, 3, 0.2);
```

Comment:
So i believe i already have a RGBA for transparent card background for the Pricing page. let's condense the two into one theme variable: `darkClearBg: {rgba value};`

Reply:
and replace the RGBA in both spots with your new theme var

Reply:
the pricing page was a little diff color . it was rgba(0, 0, 0, 0.3) and the new one is rgba(3,3,3,0.2)

Reply:
That's fine. basically was saying to normalize both to use the same, whichever one you like better + theme variable it